### ISSUE TITLE
License and requirements

### ISSUE THREAD

**Issue by afuetterer** [PetrKorab/Animated-Word-Cloud-in-Economics#1]
Hi @PetrKorab,

I enjoyed your article on Python in Plain English on this. The original WordSwarm project has no recent activity, so I am asking here: What is the license of this project? The original WordSwarm is licensed under GPL, so I guess your project is GPL as well?
Could you add a requirements.txt and/or install instructions on how to get this project to run?

Thank you.

**Comment by PetrKorab** [comment]
Hi @afuetterer,

I will add all requested files. Also, I will update the readme to make it easier for you to use it.

Please give me a couple of weeks for these updates. Thanks.

**Comment by afuetterer** [comment]
Thank you very much for the kind reply. Looking forward to it.
